### DISCUSSION TITLE
Write a short and descriptive title!

### DISCUSSION THREAD

**Discussion by zylthinking** [typst/typst#4670] [General]
### Description

the documents says counter.get() returns a int arrary, but indeed currently it returns a content. Which make it impossible to compare like the following:

```
            let p = context counter(page).get()
            if p.at(0) > 1 {
                set text(size:5pt)
                title + h(1fr) + counter(page).display("1", both: false);
                v(-1em) + line(length: 100%, stroke: 0.3pt);					
            }
``` 

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by xkevio** [comment]
`counter.get()` does not return `content`. Anything inside a `context`-expression does. You have to evaluate your context-dependent variables all inside a `context {}` block.
